File: app/src/main/java/com/example/usermanagmentapp/ui/main/UserViewHolder.java
package com.example.usermanagmentapp.ui.main;

import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.example.usermanagmentapp.databinding.RowUserBinding;

public class UserViewHolder extends RecyclerView.ViewHolder {
    public final TextView userTextView;
    public final ImageView imageView;

    public UserViewHolder(@NonNull RowUserBinding binding) {
        super(binding.getRoot());
        userTextView = binding.userTextView;
        imageView = binding.imageView;
    }
}

File: app/src/main/java/com/example/usermanagmentapp/ui/main/UserViewHolder.kt
package com.example.usermanagmentapp.ui.main

import androidx.recyclerview.widget.RecyclerView.ViewHolder
import com.example.usermanagmentapp.databinding.RowUserBinding

class UserViewHolder(private val binding: RowUserBinding) : ViewHolder(binding.root) {
    val userTextView = binding.userTextView
    val imageView = binding.imageView
}